cfme/tests/networks/test_sdn_balancers.py

```diff
@@ -2,21 +2,22 @@
 
 from cfme.cloud.provider.azure import AzureProvider
 from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.cloud.provider.gce import GCEProvider
 from cfme.cloud.provider.openstack import OpenStackProvider
 from cfme.exceptions import DestinationNotFound
 from cfme.utils.appliance.implementations.ui import navigate_to
-
+from cfme.utils.blockers import BZ
 
 pytestmark = [
     pytest.mark.usefixtures('setup_provider'),
-    pytest.mark.provider([AzureProvider, EC2Provider, OpenStackProvider],
+    pytest.mark.provider([AzureProvider, EC2Provider, OpenStackProvider, GCEProvider],
                          scope='module')
 ]
 
 
 @pytest.fixture(scope='module')
-def network_prov_with_load_balancers(appliance):
-    prov_collection = appliance.collections.network_providers
+def network_prov_with_load_balancers(provider):
+    prov_collection = provider.appliance.collections.network_providers
```

Comment:
**QUESTION**
It's not clear why did you change it?

Reply:
Because with appliance, the fixture was setup only once and never changing, instead of for each provider